CONTRIBUTING.md

```diff
@@ -312,6 +312,51 @@ load the data. Right now we don't provide any instructions inside here yet, but
 [here's how the official ReasonML playground did
 it](https://github.com/reasonml/reasonml.github.io/blob/source/website/setupSomeArtifacts.js#L65).
 
+### Upgrading the Reason version
+
+The playground is shipping with a specific Reason syntax version, which currently needs to be updated manually once in a while.
+Paths to relevant files:
+
+- `jscomp/main/refmt_api.ml`: Contains the programmatic interface to the Reason syntax converter (responsible for transforming Reason string code to an OCaml AST)
+- `lib/4.06.1/refmt_main3.ml`: The Reason binary used within BuckleScript itself. The `3` corresponds to the major version number of Reason.
+
+Both files are generated by using the `jscomp/bin/bspack.exe` binary (which is also built automatically when you build the compiler inside this repository) on the Reason parser. What does that mean?
+`bspack` resolves all dependencies of one specific `.ml` input file, orders them by according to the dependency tree and then copies all dependencies including the input file in one huge `.ml` file.
+
+So the two files mentioned above, `refmt_api.ml` and `refmt_main3.ml`, are bspacked within the Reason repository and then checked into the BuckleScript repository (we call this `vendoring` or `snapshotting`).
+Here are the instructions on building your own Reason snapshots (make sure you to have everything set up for building the playground bundle first, as mentioned above):
+
+```
+# Let's go up one level and clone Reason in a sibling directory next to your `bucklescript` repo
+cd ..
+git clone https://github.com/facebook/reason
+
+cd reason
```

Comment:
You're missing `opam switch create 4.06.1` / `opam switch 4.06.1` . This is a very important step

Reply:
Right before the codeblock it says: `(make sure you to have everything set up for building the playground bundle first, as mentioned above)` ... so you should have a 4.06.1 switch by now?

Reply:
added some extra note